dev-support/create-release/release-util.sh

```diff
@@ -304,6 +306,10 @@ function check_needed_vars {
   return 0
 }
 
+function is_svn_ci_disabled {
+  [[ "$DRY_RUN" = 1 ]] || [[ "$IGNORE_SVN_CI" = 1 ]]
```

Comment:
why can't we just use `dry-run`/ no `-f` ? seems like they're doing the same thing, please correct me if I'm missing something.

Reply:
> is this change for using -ignore-svn-ci with -f together?

That's true, `ignore-svn-ci` is only useful when used in combination with `-f`, otherwise it is sort of no-op because dry-mode anyways don't perform svn ci.